content/concepts/discovery-routing/mDNS.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "mDNS"
+description: "mDNS uses a multicast system of DNS records over a local network to enable peer discovery."
+weight: 224
+---
+
+## What is mDNS?
+
+mDNS, or multicast Domain Name System, is a way for nodes to use a multicast system
+of DNS records over a local network to discover and communicate with nodes. Nodes
+broadcast topics they're interested in instead of querying a central name server.
+The discovery, however, is limited to the peers in the local network. mDNS is commonly
+used on home networks to allow devices such as computers, printers, and smart TVs to
+find each other and connect. It uses a protocol called multicast to broadcast messages
+on the network, allowing devices to discover each other and exchange information.
+
+## mDNS in libp2p
+
+In libp2p, mDNS is used for peer discovery, allowing peers to find and
+communicate with each other on the same local network without any prior configuration.
+This is achieved through multicast DNS (mDNS) records, which are sent to all nodes on the
+local network.
+
+To initiate peer discovery, a peer sends a query to all other peers on the network using
+a DNS message with the question `_p2p._udp.local PTR`. In response, each peer will send a
```

Comment:
The question is for `_p2p` (or is it `p2p`?) The rest is just there to make things fit into mDNS.

Reply:
it's for _p2p._udp.local.
<img width="759" alt="Screenshot 2023-01-26 at 4 37 04 PM" src="https://user-images.githubusercontent.com/594035/214981290-c65f7d31-b102-4270-860c-9b6555b870ed.png">

I think this is to low level. If I need to use wireshark to verify this, I don't think users will care about the specifics here.